### PR TITLE
add draft 1.44 release announcement

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

**Comment by link2xt** [comment]
The post is not visible in the preview for some reason

**Comment by Simon-Laux** [comment]
iOS notifications in general did also receive fixes, so it is somewhat more working even for normal non chatmail accounts.

File: _posts/2024-03-11-jumbo44.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+The 1.44 releases on all stores and platforms bring
+several often-requested improvements.
+We keep talking about each one only briefly because there are so many :) 
+
+## Reactions ❤️  now supported on all platforms 
+
+![Select reactions to chat message](../assets/blog/desktop-reactions.png)
+
+Desktop, iOS and UbuntuTouch apps can now send arbitrary reactions 
+on chat messages and surpassed Android support
```
Comment (Simon-Laux) [inline _posts/2024-03-11-jumbo44.md]: were reactions on android non experimental before 1.44? if not, why de-emphasize it soo much here?

File: _posts/2024-03-11-jumbo44.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## New Desktop Multi-account sidebar
+
+![Desktop multi-account sidebar](../assets/blog/desktop-sidebar.jpg)
+
+The Desktop apps on all platforms now have a sidebar
+that shows and allows to select all your accounts,
+and eases a direct overview of unread messages in any accounts. 
+Moreover, system notifications now work, including on Flathub 
+where severe system integration bugs got fixed.
```
Comment (Simon-Laux) [inline _posts/2024-03-11-jumbo44.md]:
```suggestion
System notifications now work for all accounts and you can also mute less important accounts.
We also fixed many issues, including the Flathub release (which crashed when receiving messages).
```

File: _posts/2024-03-11-jumbo44.md (hunk 1 above)
Comment (hpk42) [reply _posts/2024-03-11-jumbo44.md]: changed it.

**Comment by hpk42** [comment]
> iOS notifications in general did also receive fixes, so it is somewhat more working even for normal non chatmail accounts.

added a sentence.

File: _posts/2024-03-11-jumbo44.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- Creating contacts and changing your avatar now syncs to all your devices. 
+
+- Many bug fixes 
+
+For complete lists of changes see: 
+
+- [Desktop changes](https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md) 
+- [Android changes](https://github.com/deltachat/deltachat-android/blob/master/CHANGELOG.md) 
+- [iOS changes](https://github.com/deltachat/deltachat-ios/blob/master/CHANGELOG.md) 
+- [Core Rust changes](https://github.com/deltachat/deltachat-core-rust/blob/main/CHANGELOG.md)
```
Comment (missytake) [inline _posts/2024-03-11-jumbo44.md]:
```suggestion
- [Core Rust changes](https://github.com/deltachat/deltachat-core-rust/blob/main/CHANGELOG.md) 
- [Ubuntu Touch changes](https://codeberg.org/lk108/deltatouch/src/branch/main/CHANGELOG)
```

**Review by missytake — APPROVED** [review]
Nice! Great to see how much happened in the past months.

Maybe some are interested in the Ubuntu Touch changelog as well, even though it's not super pretty:

File: _posts/2024-03-11-jumbo44.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- Improved Desktop "settings" dialog and many other Desktop and iOS UI refinements 
+
+- Creating contacts and changing your avatar now syncs to all your devices. 
+
+- Many bug fixes 
+
+For complete lists of changes see: 
+
+- [Desktop changes](https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md) 
+- [Android changes](https://github.com/deltachat/deltachat-android/blob/master/CHANGELOG.md)
```
Comment (r10s) [inline _posts/2024-03-11-jumbo44.md]:
```suggestion
- [Android changes](https://github.com/deltachat/deltachat-android/blob/main/CHANGELOG.md) 
```

master is somehow stuck in the past at 1.40 or so

File: _posts/2024-03-11-jumbo44.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+- Creating contacts and changing your avatar now syncs to all your devices. 
+
+- Many bug fixes 
+
+For complete lists of changes see: 
+
+- [Desktop changes](https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md) 
+- [Android changes](https://github.com/deltachat/deltachat-android/blob/master/CHANGELOG.md) 
+- [iOS changes](https://github.com/deltachat/deltachat-ios/blob/master/CHANGELOG.md)
```
Comment (r10s) [inline _posts/2024-03-11-jumbo44.md]:
```suggestion
- [iOS changes](https://github.com/deltachat/deltachat-ios/blob/main/CHANGELOG.md) 
```

**Pull request merged**